### DISCUSSION TITLE
so it turns out talk-llama can just take over your computer

### DISCUSSION THREAD

**Discussion by alexpeys** [ggml-org/whisper.cpp#1342] [Show and tell]
It looks like if i ask talk llama with wizard coder to write a python script when the tts goes back to read the ```python line it actually executes the line and opens python... 

<img width="800" alt="image" src="https://github.com/ggerganov/whisper.cpp/assets/5770141/7250259a-8dee-4b8c-aa6c-683049bade7b">

also wizard coder is very mean, see the lines before, but this is not related to the whisper.cpp issue

**Comment by ggerganov** [comment]
It's trying to escape 🤣
